src/components/views/FilesDropdown.tsx

```diff
@@ -83,6 +87,63 @@ const isFileValidForFigure = (
   }
 }
 
+const getFigures = (doc: ProsemirrorNode) => {
+  const _figures = new Map<string, string>()
+  doc.descendants((node) => {
+    if (node.type === schema.nodes.figure) {
+      const _id = node.attrs.src.replace('attachment:', '')
+      _figures.set(_id, _id)
+    }
+  })
+  return _figures
+}
+
+// Get Supplement files from modelMap
+const getSupplementFiles = (
+  modelMap: Map<string, Model>,
+  attachments: FileAttachment[],
+  filePredicate?: FilePredicate
```

Comment:
Can you please rename this parameter and type to something more comprehensive? The name tells me only that it returns a boolean.